Microsoft Teams: Just a heads up - Adding new actions to Microsoft Teams SOAR apps (#20)
splunk-soar-connectors/microsoftteams: manual_readme_content.md

[comment]: # "either express or implied. See the License for the specific language governing permissions"
[comment]: # "and limitations under the License."
[comment]: # ""
## Playbook Backward Compatibility

> - The existing action 'send message' has been renamed to 'send channel message'. Hence, it is requested the end-user to please update their existing playbooks by  modifying the action name accordingly.

Reviewer: Please add version information so that it's clear when this is applicable

> - With version 3.0.0 of the connector,The action 'send message' has been renamed to 'send channel message'.Therefore, we request that end-users please update their existing playbooks by modifying the action name accordingly.
> 
>      - Update the 'send message' action name to 'send channel message'.

Reviewer: ```suggestion
- With version 3.0.0 of the connector, the 'send message' action has been renamed to 'send channel message'. Please update any existing playbooks by modifying the action name accordingly.
```
